README.md

```diff
@@ -134,8 +134,8 @@ Implementations can subclass `opentracing.Trace`, `opentracing.Span`, and the ot
 If `mocha` is being used for unit testing, `test/api_compatibility` can be used to test the custom tracer. The file exports a single function that expects as an argument a function that will return a new instance of the tracer.
 
 ```javascript
-const apiCompatibilityChecks = require('opentracing/lib/test/api_compatibility.js');
-apiCompatibilityCheck(() => new CustomTracer());
+const { apiCompatibilityChecks } = require('opentracing/lib/test/api_compatibility.js');
+apiCompatibilityChecks(() => new CustomTracer());
```

Comment:
People claim it does not work without `.default` in some later versions (of Babel?). I am not an expert in JS ecosystem. I will merge as is, you may want to reply to that other ticket.